### ISSUE TITLE
`/api/scm/info` incorrect for sites configured to deploy from GitHub 

### ISSUE THREAD

**Issue by duncanmak** [projectkudu/kudu#2626]
I have Azure AppServices configured to deploy from GitHub commits.

When I use the `/api/scm/info` REST endpoint, I get back `$sitename.scm.azurewebsites.net/$sitename.git`, not where I have it configured on GitHub.

**Comment by davidebbo** [comment]
That API is kind of quirky. To git the GitHub info, call this ARM API instead:

    /subscriptions/{sub}/resourceGroups/{rg}/providers/Microsoft.Web/sites/{site}/sourcecontrols/web?api-version=2016-08-01

You can play with it easily using https://resources.azure.com/

**Comment by duncanmak** [comment]
@davidebbo Thanks for the quick reply.

I'll go learn how to use the ARM API, but it'd be nice if `/api/scm/info` also offered that info.

In fact, I came across this because I'm writing a thing to process the Kudu PostDeployment WebHook. I noticed that the `id` field is populated with the git commit hash - if the origin git repo location can be surfaced in the Post Deployment JSON payload, that'd really be the best. 

Perhaps `id` could be made to be the location to the commit, i.e. `github.com/:org/:repo/commit/:id` instead of just `id`?
